docs/cryoet_data_portal_docsite_data.md

```diff
@@ -25,7 +25,43 @@ Datasets are contributed sets of image files associated with imaging one sample
 
 The Browse Datasets page shows a table of all datasets on the Portal. These datasets are not currently ordered. Instead, the left side filter panel provides options for filtering the table according to files included in the datasets, such as ground truth annotation files; the author or ID of the dataset; organism in the sample; hardware; metadata for the tilt series or reconstructed tomograms. In addition, the search bar filters based on keywords or phrases contained in the dataset titles. The dataset entries in the table have descriptive names, such as "S. pombe cryo-FIB lamellae acquired with defocus-only," which aim to summarize the experiment as well as a Dataset ID assigned by the Portal, the organism name, number of runs in the dataset, and list of annotated objects, such as membrane. Datasets on the Portal may be found in other image databases. On the Browse Datasets page, the datasets table shows the EMPIAR ID for datasets that are also found on the Electron Microscopy Public Image Archive.
 
-On a given Dataset Overview page, the View All Info panel contains metadata for the dataset.
+On a given Dataset Overview page, the View All Info panel contains metadata for the dataset. These metadata are defined in the tables below including their mapping to attributes in the Portal API:
+
+**Dataset Metadata**
+| **Portal Metadata** | **API Expression**                    | **Definition**                                                      |
+|---------------------|---------------------------------------|---------------------------------------------------------------------|
+| Deposition Date     | Dataset.deposition_date               | Date when a dataset is initially received by the Data Portal.       |
+| Grant ID            | DatasetFunding.grant_id               | Grant identifier provided by the funding agency.                    |
+| Funding Agency      | DatasetFunding.funding_agency_name    | Name of the funding agency.                                         |
+| Related Databases   | Dataset.related_database_entries      | The dataset identifier for other databases, e.g. EMPIAR, that contain this dataset. |
+| Citations           | Dataset.dataset_citations             | DOIs for publications that cite the dataset.                        |
```

Comment:
`Citations` will be removed from the schema and is not currently populated for any dataset, so we may want to remove it here.